__tests__/app.test.js

```diff
@@ -75,8 +75,23 @@ describe("GET /api/articles/:article_id", () => {
         expect(article.author).toBe("icellusedkars");
         expect(article.topic).toBe("mitch");
         expect(article.article_id).toBe(2);
+        expect(article.comment_count).toBe("0")
```

Comment:
we dont have to test this in both tests - once is fine and we shouldn't have to go back and alter previous tests